readthedocs/api/v3/views.py

```diff
@@ -680,9 +680,13 @@ class NotificationsOrganizationViewSet(
 
     def get_queryset(self):
         content_type = ContentType.objects.get_for_model(Organization)
-        return self.queryset.filter(
-            attached_to_content_type=content_type,
-            attached_to_id__in=AdminPermission.organizations(
-                self.request.user, owner=True, member=False
-            ).values("id"),
+        return (
+            super()
+            .get_queryset()
+            .filter(
+                attached_to_content_type=content_type,
+                attached_to_id__in=AdminPermission.organizations(
+                    self.request.user, owner=True, member=False
+                ).values("id"),
```

Comment:
I believe you don't need this, since it should already be filtered from `for_user`

https://github.com/readthedocs/readthedocs.org/blob/ea4337133c6d5c07260aad819541fbb9afc3ca24/readthedocs/notifications/querysets.py#L100-L107

```suggestion
```

Reply:
Or wonder if we can just use something similar to

https://github.com/readthedocs/readthedocs.org/blob/5245ebbc35f9d93f30f47fed51ced4c829adedd2/readthedocs/api/v3/views.py#L446-L448

Use `_get_parent_organization`

Reply:
Oh, nice catch, I am not quite familiar with this part of the codebase :)